### DISCUSSION TITLE
Does this package support Expo(EAS)

### DISCUSSION THREAD

**Discussion by leandrosimoes** [leandrosimoes/react-native-android-notification-listener#29] [Q&A]
Opened because of the issue https://github.com/leandrosimoes/react-native-android-notification-listener/issues/28

**Comment by leandrosimoes** [comment]
I couldn't find anything that says that it wouldn't work. As you can see in the official [EAS Build](https://docs.expo.dev/build/introduction/) docs, it should work with any Expo or React Native application.

**Reply by matheusf31** [reply]
Hello, I follow the example code and when I console the RNAndroidNotificationListener in expo project that returns null...

expo version: "expo": "~48.0.18"

**Reply by leandrosimoes** [reply]
Thanks @matheusf31,

I already opened an issue based on your answer here. It would be nice if you could help us with that since I don't have much experience with expo projects.

Thanks 😄

**Reply by Ademola101** [reply]
It works for me
